packages/workbox-webpack-plugin/src/index.js

```diff
@@ -128,33 +131,26 @@ class WorkboxWebpackPlugin {
      *     assets.
      */
     compiler.plugin('emit', async (compilation, next) => {
-      const {
-        manifestFilename = `file-manifest.${compilation.hash}.js`,
-        swSrc,
-        filename = (swSrc && path.basename(swSrc)) || 'sw.js',
-      } = this.config;
+      const {swSrc} = this.config;
+      const serviceWorkerFilename = swSrc ? path.basename(swSrc) : 'sw.js';
 
-      const entries = getEntries(compiler, compilation, this.config);
+      const entries = getEntries(compilation, this.config);
+
+      const fileManifest = generateManifest(entries);
+      const fileManifestAsset = webpackAsset(fileManifest);
```

Comment:
Nit: Would is be possible to rename this? "webpackAsset" doesn't really explain what it's doing.

Reply:
I understand what Jeff's referring to here, but agree that either clearer naming for contributors who aren't as familiar with webpack or a comment to clarify would be useful.